### ISSUE TITLE
Site has custom !DOCTYPE

### ISSUE THREAD

**Issue by DioxusGrow** [James-LG/Skyscraper#32]
How to fix Err` value: MissingTagCloseAfterDoctype?
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="de" lang="de" xmlns:og="http://ogp.me/ns#" xmlns:fb="http://www.facebook.com/2008/fbml" >

</html>
```

**Comment by James-LG** [comment]
Hi, looks like this is actually a supported declaration used by older documents, so I will fix this.

https://www.w3schools.com/tags/tag_doctype.asp

**Comment by James-LG** [comment]
Fixed in [v0.6.4](https://github.com/James-LG/Skyscraper/releases/tag/v0.6.4)
